tests/_site/apps/catalogue/migrations/0016_auto_20190327_0757.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration doesn't appear to be related to the change you're making here?

Reply:
It is a dependency for `0017_auto_20190816_0938`, which has also been copied from the parent `catalogue` app.

Should I copy it over in a separate PR? I also see a couple of other forked apps whose migrations do not match those in their parent apps?

Reply:
Yes please